Kickstarter-iOS/Views/Controllers/MessageDialogViewController.swift

```diff
@@ -32,6 +33,9 @@ internal final class MessageDialogViewController: UIViewController {
   internal override func viewDidLoad() {
     super.viewDidLoad()
     self.viewModel.inputs.viewDidLoad()
+
+    _ = self.cancelButton
+      |> UIBarButtonItem.lens.title %~ { _ in Strings.general_navigation_buttons_cancel() }
```

Comment:
Is there any reason why we are not setting the title within the <bindStyles()> function?

Reply:
Oh true! I didn't really think about it... I can move it to `bindStyles()`. I guess to me a text isn't really a style but better to keep it consistent I agree 👍